pkg/notifiers/sender.go

```diff
@@ -325,6 +343,25 @@ func smtpInit(host string, port int) (*smtp.Client, error) {
 	return c, nil
 }
 
+func dialerWithTLSConfig(dialer dialer.Dialer, host, smartHost string) dialer.Dialer {
+	tlsConfig := &tls.Config{
+		ServerName: host,
```

Comment:
is that the only default we have to set in tlsconfig? anything related to client cert verification? Like https://github.com/rancher/rancher/blob/master/pkg/clustermanager/manager.go#L279